adapters/verizonmedia/verizonmedia.go

```diff
@@ -0,0 +1,189 @@
+	if err != nil {
+		err = &errortypes.BadInput{
+			Message: "ext.bidder not provided",
+		}
+		errors = append(errors, err)
+		return nil, errors
+	}
+	var verizonMediaExt openrtb_ext.ExtImpVerizonMedia
+	err = json.Unmarshal(bidderExt.Bidder, &verizonMediaExt)
+	if err != nil {
+		err = &errortypes.BadInput{
+			Message: err.Error(),
+		}
+		errors = append(errors, err)
+		return nil, errors
+	}
+
+	if verizonMediaExt.Dcn == "" {
+		err = &errortypes.BadInput{
+			Message: "Missing param dcn",
+		}
+		errors = append(errors, err)
+		return nil, errors
+	}
+
+	if verizonMediaExt.Pos == "" {
+		err = &errortypes.BadInput{
+			Message: "Missing param pos",
+		}
+		errors = append(errors, err)
+		return nil, errors
+	}
+
+	siteCopy := *request.Site
+	request.Site = &siteCopy
+	changeRequestForBidService(request, &verizonMediaExt)
+	reqJSON, err := json.Marshal(request)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	thisURI := a.URI
+
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	headers.Add("Accept", "application/json")
+	headers.Set("User-Agent", request.Device.UA)
+	headers.Add("x-openrtb-version", "2.5")
+
+	return []*adapters.RequestData{{
+		Method:  "POST",
+		Uri:     thisURI,
+		Body:    reqJSON,
+		Headers: headers,
+	}}, errors
+}
+
+func (a *VerizonMediaAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+
+	if response.StatusCode == http.StatusNoContent {
+		return nil, nil
+	}
+
+	if response.StatusCode != http.StatusOK {
+		return nil, []error{&errortypes.BadServerResponse{
+			Message: fmt.Sprintf("Unexpected status code: %d. ", response.StatusCode),
+		}}
+	}
+
+	var bidResp openrtb.BidResponse
+	if err := json.Unmarshal(response.Body, &bidResp); err != nil {
+		return nil, []error{&errortypes.BadServerResponse{
+			Message: fmt.Sprintf("Bad server response: %d. ", err),
+		}}
+	}
+
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(1)
+
+	for _, sb := range bidResp.SeatBid {
+		for _, bid := range sb.Bid {
+			if !impIDExists(bid.ImpID, internalRequest.Imp) {
+				return nil, []error{&errortypes.BadServerResponse{
+					Message: fmt.Sprintf("Unknown ad unit code '%s'", bid.ImpID),
+				}}
+			}
+
+			if openrtb_ext.BidTypeBanner != getMediaTypeForImp(bid.ImpID, internalRequest.Imp) {
+				//only banner is supported, anything else is ignored
+				continue
+			}
+
+			bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
+				Bid:     &bid,
+				BidType: openrtb_ext.BidTypeBanner,
+			})
+		}
+	}
+
+	return bidResponse, nil
+}
+
+func getMediaTypeForImp(impId string, imps []openrtb.Imp) openrtb_ext.BidType {
+	var mediaType openrtb_ext.BidType
+	for _, imp := range imps {
+		if imp.ID == impId {
+			if imp.Banner != nil {
+				mediaType = openrtb_ext.BidTypeBanner
+			}
+			return mediaType
+		}
+	}
+	return mediaType
```

Comment:
And then just return `nil` here, that would give the same result as you currently have.

Reply:
Not in scope anymore as the function is changed